controllers/kokumetricsconfig_controller.go

```diff
@@ -50,6 +50,8 @@ var (
 	pullSecretAuthKey        = "cloud.openshift.com"
 	authSecretUserKey        = "username"
 	authSecretPasswordKey    = "password"
+	authClientId             = "clientid"
+	authClientSecret         = "clientsecret"
```

Comment:
Thinking about this some more, when we make the curl request, we're using `client_id` and `client_secret`. I think we should use those here and make customers create their secret using `client_id` and `client_secret` to match how a curl request would be done.

Reply:
sounds reasonable 👍🏾